src/formatted-text/FormattedText.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { props, t, skinnable } from '../utils';
+import flattenDeep from 'lodash/flattenDeep';
+import tlds from 'tlds';
+
+const linkify = require('linkify-it')().tlds(tlds);
+
+export const Props = {
+  content: t.String,
+  id: t.maybe(t.String),
+  className: t.maybe(t.String),
+  style: t.maybe(t.Object)
+};
+
+/** A component that transforms `\n` in `<br>` and links in `<a>`
+ * @param content - The content of the paragraph
+ * @param paragraphSpacing - The space occupied by an empty line
+ */
+@skinnable()
+@props(Props)
+export default class FormattedText extends React.Component {
+
+  linkify(string) {
+    const matches = linkify.match(string);
+    if (matches) {
+      return matches.reduce((acc, match, i) => {
+        const nextMatch = matches[i + 1];
+
+        return [
+          ...(acc || [string.slice(0, match.index)]),
+          <a href={match.url} target='_blank'>{match.raw}</a>,
+          string.slice(match.lastIndex, nextMatch ? nextMatch.index : undefined)
+        ];
+      }, null);
```

Comment:
I'm having quite an hard time reading this `reduce`, maybe `matches.reduce((s, match) => s.replace(match.raw, 'LINK'), initialString)` would be simpler?
but not sure, feel free to ignore